# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/50_mm-2.7-linux.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mm2.7/src/config.h.linux	Sun Jun 18 21:59:38 2000
+++ mm2.7/src/config.h	Sun Jun 18 21:59:10 2000
@@ -100,9 +100,11 @@
 #endif
 
 #ifdef SYSV
-#define killpg(a, b) kill(-(a), (b))
-#define bcopy(a, b, c) memcpy(b, a, c)
-#define bzero(a, b) memset(a, 0, b)
+#ifndef linux
+# define killpg(a, b) kill(-(a), (b))
+# define bcopy(a, b, c) memcpy(b, a, c)
+# define bzero(a, b) memset(a, 0, b)
+#endif
 #define bcmp memcmp
 #define index strchr
 #define rindex strrchr
